server/controllers/index.js

```diff
@@ -155,11 +194,18 @@ router.post('/api/voteCard', auth, (req, res) => {
 /* Check if in dev mode, and enable end game request */
 if (process.env.NODE_ENV === 'testing') {
     router.post('/api/reset-server', (req, res) => {
-        gameLogic.clearGameState();
-        closeSocket();
-        req.session.destroy();
-        currentUsers = [];
-        res.sendStatus(200);
+        try {
+            if (req.session.roomId) {
```

Comment:
This should probably delete all rooms right? Are you keeping a list anywhere where we can loop through and clear them all

Reply:
Yeah the socket file takes care of this because it stores the list of rooms. Although I just remembered I'm not setting the roomId back to 0, so thanks